rdmo/projects/templates/projects/project_form.html

```diff
@@ -31,6 +31,23 @@ <h1>{% trans 'Update project tasks' %}</h1>
         <h1>{% trans 'Update project views' %}</h1>
         {% bootstrap_form submit=_('Save views') %}
 
+    {% elif request.resolver_match.url_name == 'project_update_visibility' %}
+
+        <h1>{% trans 'Update project visibility' %}</h1>
+
+        <p class="mb-20">
+            {% blocktrans trimmed %}
+            Projects can be made visible to all users, for example to be used as a template.
+            When a project is made visible, users can access it as if they were in the guest role.
+            {% endblocktrans %}
+        </p>
+
+        {% if object.visibility %}
+        {% bootstrap_form submit=_('Save visibility') delete=_('Remove visibility') %}
```

Comment:
in the multisite, I don't find the 'Remove visibility' button?

Reply:
I created a separate form for the visibility.